lib/YaST/NetworkSettings/NetworkCardSetup/BondSlavesTab.pm

```diff
@@ -35,8 +36,10 @@ sub select_tab {
 
 sub select_bond_slave_in_list {
     assert_screen(BOND_SLAVES_TAB);
-    record_soft_failure('bsc#1191112 - Resizing window as workaround for YaST content not loading');
-    send_key_until_needlematch(BOND_SLAVE_DEVICE_CHECKBOX_UNCHECKED, 'alt-f10', 10, 2);
+    if (!is_sle('=15-SP5')) {
```

Comment:
I guess we can use is_sle('>=15-SP5')

Reply:
If thinking for later service pack, we need set (!is_sle('>=15-SP5')), just not sure how long need we keep the workaround for old version.

Reply:
Just for simplicity we could go for 
```suggestion
    if (is_sle('=15-SP4')) {
```